.git-commit-template

```diff
@@ -0,0 +1,21 @@
+# Subject line - 50 chars (for log --oneline)
+# ============================================== #
+
```

Comment:
The subject line can be 72 chars for GitHub.

The body can be as long as you want and using hard returns doesn't matter.

Like the concept though.